playlet-lib/src/components/VideoPlayer/VideoPlayer.bs

```diff
@@ -244,6 +255,51 @@ function OnVideoFinished() as void
     end if
 end function
 
+function ShowUIAndStartTimer()
+    m.top.showUI = { "show": true, "trickPlaybar": true, "titleAndTime": true, "trickplayBackgroundOverlay": true, "focusTrickplayBar": false }
```

Comment:
Can you add a check to see if the `showUI` field exists? Since it is not documented, we don't know at which Roku OS version it was introduced, or if it might be removed.
Something like `if m.top.hasField("showUI")`

Reply:
Yep, thank you, fixed that.